.github/CODEOWNERS

```diff
@@ -25,4 +25,6 @@ yarn.lock                    @janus-idp/maintainers-plugins
 /plugins/quay/               @janus-idp/maintainers-plugins @fmenesesg
 /plugins/topology/           @janus-idp/maintainers-plugins @debsmita1 @divyanshiGupta
 /plugins/tekton/             @janus-idp/maintainers-plugins @debsmita1 @divyanshiGupta
+/plugins/rbac-backend/       @janus-idp/maintainers-plugins @gorkem @AndrienkoAleksandr
```

Comment:
Could you apply @PatAKnight for these plugins too?

Reply:
Done 🙂